Sources/Transmute/Error.swift

```diff
@@ -19,10 +19,22 @@ extension Package {
 
     public enum InvalidLayoutType {
         case MultipleSourceRoots([String])
-        case InvalidLayout
+        case InvalidLayout([String])
     }
 }
 
+extension Package.InvalidLayoutType: CustomStringConvertible {
+    public var description: String {
+        switch self {
+        case .MultipleSourceRoots(let paths):
```

Comment:
The enum case was already there. It isn't used elsewhere in the SwiftPM code. I guess it could be removed. That would leave the InvalidLayoutType with only a single member; it could even be merged into ModuleError altogether, like this:

    public enum ModuleError: ErrorProtocol {
        case NoModules(Package)
        case ModuleNotFound(String)
        case InvalidLayout([String])
    }